ta-sdk-spi/src/main/java/com/ibm/ta/sdk/spi/validation/TaCollectionZipValidator.java

```diff
@@ -6,26 +6,152 @@
 
 package com.ibm.ta.sdk.spi.validation;
 
+import com.google.gson.Gson;
+import com.ibm.ta.sdk.spi.collect.EnvironmentJson;
+import com.ibm.ta.sdk.spi.plugin.TADataCollector;
+
+import java.io.ByteArrayInputStream;
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.*;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
 
 public class TaCollectionZipValidator {
-
-    private static final String FILE_XML = ".xml";
-    private static final String FILE_JSON = ".json";
-    private static final String FILE_LOG = ".log";
     private static final String FILE_ZIP = ".zip";
 
-    public static boolean validateCollection(String zipFilePath){
-        boolean isValid = true;
+    public static boolean validateCollectionArchive(String zipFilePath) {
         if (!zipFilePath.endsWith(FILE_ZIP)) {
-            System.out.println("Input collection file path is not a zip file");
-            isValid = false;
+            System.err.println("Input collection archive file is not a zip file");
```

Comment:
Are we using System.out/err everywhere? I thought we had a logger in the SDK. I might be wrong